xmr-btc/src/config.rs

```diff
@@ -35,15 +38,21 @@ mod mainnet {
     pub static BITCOIN_FINALITY_CONFIRMATIONS: u32 = 3;
 
     pub static BITCOIN_AVG_BLOCK_TIME: Lazy<Duration> = Lazy::new(|| Duration::from_secs(10 * 60));
+
+    pub static MONERO_MAX_FINALITY_TIME: Lazy<Duration> =
+        Lazy::new(|| Duration::from_secs_f64(15f64 * 1.5 * 2f64 * 60f64));
```

Comment:
It'd be clearer if you use 2 vars: one for average block time and one for number of confirmations and then provide a function that does the calculation for the total time.